gnome-base/nautilus/files/43.0-fix-clang-build.patch

```diff
@@ -0,0 +1,30 @@
+From 68b050ac0f60b1f02767276643cba8eb701f9884 Mon Sep 17 00:00:00 2001
```

Comment:
Is this upstream somewhere? If not, could you submit it please & put a link in the patch? Thanks!